.circleci/config.yml

```diff
@@ -288,7 +288,7 @@ workflows:
   build:
     when: << pipeline.parameters.run_default_workflow >>
     jobs:
-      - unit-php74-magento24
+      #- unit-php74-magento24
```

Comment:
This is just waiting on @dumega's changes to get merged in now basically, right?

Reply:
ya